cmake/tribits/common_tpls/FindTPLNetcdf.cmake

```diff
@@ -85,10 +85,16 @@ IF (Netcdf_ALLOW_PREFIND)
       "${DOCSTR} header files.")
   ENDIF()
 ELSE()
-  # Curl library is only required if DAP is enabled; should detect inside FindNetCDF.cmake,
-  # but that is not being called... SEMS has DAP enabled; many HPC systems don't, but they
-  # override these settings...
-  SET(REQUIRED_LIBS_NAMES ${REQUIRED_LIBS_NAMES} curl)
+  # Curl library is only required if DAP is enabled; should detect inside
+  # FindNetCDF.cmake, but that is not being called... SEMS has DAP enabled;
+  # many HPC systems don't, but they override these settings...
+  EXECUTE_PROCESS(COMMAND "nc-config --has-dap2"
```

Comment:
@dc-snl: Perhaps check for existence of nc-config. If it doesn't exist, `SET(REQUIRED_LIBS_NAMES ${REQUIRED_LIBS_NAMES} curl)`.

Reply:
Good idea. Testing this out now.

Reply:
Finally got a chance to test this out and push. Seems to work on SEMS and Vortex. Testing on Astra now.

Reply:
Hang on. It doesn't work on Vortex. I'll look into this tomorrow.

Reply:
Ah, okay. Forgot to wipe cmake. So building Trilinos on Vortex, Stria, and SEMS seems to work.

Running `ctest-s-local-test-driver.sh` on Vortex now.